src/azul/time.py

```diff
@@ -114,6 +114,13 @@ def parse_http_date(http_date: str, base_time: Optional[float] = None) -> float:
 dcp2_datetime_format = '%Y-%m-%dT%H:%M:%S.%f%z'
 
 
+def optional_dcp2_datetime(d: Optional[datetime] = None) -> Optional[str]:
```

Comment:
Should probably be a method of the transformer or a function in the module containing the transformer.

Reply:
And no default value for the kwarg. It would be silly to call this without passing arguments.